tools/expotools/src/commands/UpdateReactNativeDocs.ts

```diff
@@ -0,0 +1,289 @@
+const RN_DOCS_DIR = path.join(RN_REPO_DIR, 'docs');
+
+const PREFIX_ADDED = 'ADDED_';
+const PREFIX_REMOVED = 'REMOVED_';
+const SUFFIX_CHANGED = '.diff';
+
+const DOCS_IGNORED = [
+  'appregistry',
+  'components-and-apis',
+  'drawerlayoutandroid',
+  'linking',
+  'settings',
+  'systrace',
+];
+
+async function action(input: Options) {
+  const options = await getOptions(input);
+
+  if (!await validateGitStatusAsync()) {
+    return;
+  }
+
+  await updateDocsAsync(options);
+
+  const summary = getDocsSummary(
+    await getLocalFilesAsync(options),
+    await getUpstreamFilesAsync(options),
+  );
+
+  logger.log();
+
+  await applyAddedFilesAsync(options, summary);
+  await applyChangedFilesAsync(options, summary);
+  await applyRemovedFilesAsync(options, summary);
+
+  logCompleted(options);
+}
+
+async function getOptions(input: Options): Promise<Options> {
+  const questions: inquirer.Question[] = [];
+  const existingSdks = (await fs.promises.readdir(SDK_DOCS_DIR, { withFileTypes: true }))
+      .filter(entry => entry.isDirectory() && entry.name !== 'latest')
+      .map(entry => entry.name.replace(/v([0-9]+)/, '$1'));
+
+  if (input.sdk && !existingSdks.includes(input.sdk)) {
+    throw new Error(`SDK docs ${input.sdk} does not exist, please create it with "et generate-sdk-docs"`);
+  }
+
+  if (!input.sdk) {
+    questions.push({
+      type: 'list',
+      name: 'sdk',
+      message: 'What Expo SDK version do you want to update?',
+      choices: existingSdks,
+    });
+  }
+
+  if (!input.from) {
+    questions.push({
+      type: 'input',
+      name: 'from',
+      message: 'From which commit of the React Native Website do you want to update? (e.g. 9806ddd)',
+      filter: (value: string) => value.trim(),
+      validate: (value: string) => value.length !== 0,
+    });
+  }
+
+  const answers = questions.length > 0 ? await inquirer.prompt(questions) : {};
+
+  return {
+    sdk: answers.sdk || input.sdk,
+    from: answers.from || input.from,
+    to: input.to || 'master',
+  };
+}
+
+async function validateGitStatusAsync() {
+  logger.info('\n📑 Checking local repository status...')
+
+  const result = await spawnAsync('git', ['status', '--porcelain']);
+  const status = result.stdout === '' ? 'clean' : 'dirty';
+
+  if (status === 'clean') {
+    return true;
+  }
+
+  logger.warn(`⚠️  Your git working tree is`, chalk.underline('dirty'));
+  logger.info(`It's recommended to ${chalk.bold('commit all your changes before proceeding')}, so you can revert the changes made by this command if necessary.`);
+
+  const { useDirtyGit } = await inquirer.prompt({
+    type: 'confirm',
+    name: 'useDirtyGit',
+    message: `Would you like to proceed?`,
+    default: false,
+  });
+
+  logger.log();
+
+  return useDirtyGit;
+}
+
+async function updateDocsAsync(options: Options) {
+  logger.info(`📚 Updating ${chalk.cyan('react-native-website')} submodule...`);
+
+  await spawnAsync('git', ['checkout', 'master'], { cwd: RN_REPO_DIR });
```

Comment:
Sorry that I haven't noticed this before – there is also a tool in `Git.ts` that you can use here 😅 
Just import `GitDirectory` from there and create an instance like this `new GitDirectory(RN_REPO_DIR);`.
Probably some commands needs to be added there (`checkout` for example) but it's worth to migrate it to this tool 😉

Reply:
Sure, sounds good! I'll add that, thanks for the tip.

Reply:
Done! Works pretty good 😄

Reply:
Yeah, sure! Sounds good, it's been an oversight to require it 😅